integration_tests/src/main/python/delta_lake_utils.py

```diff
@@ -31,6 +32,14 @@
     "SortExec"
 ]
 
+# Disable Deletion Vectors except for Databricks 14.3
+def deletion_vector_conf_with_reason(reason='https://github.com/NVIDIA/spark-rapids/issues/12042'):
+    return list(filter(None, ({'spark.databricks.delta.properties.defaults.enableDeletionVectors': 'false'},
+                              pytest.param({'spark.databricks.delta.properties.defaults.enableDeletionVectors': 'true'},
+                                           marks=pytest.mark.xfail(
+                                               reason=reason)) if is_databricks143_or_later() else None)))
```

Comment:
Should we have a follow on issue to verify that we fall back to the CPU for 2.4.0?

I don't want to do it now because https://docs.delta.io/latest/delta-deletion-vectors.html shows that deletion vectors were not fully supported until 3.1.0 anyways.